bindings/azure/storagequeues/storagequeues.go

```diff
@@ -178,8 +184,15 @@ func (d *AzureQueueHelper) Read(ctx context.Context, consumer *consumer) error {
 	}
 
 	_, err = consumer.callback(ctx, &bindings.ReadResponse{
-		Data:     data,
-		Metadata: map[string]string{},
+		Data: data,
+		Metadata: map[string]string{
+			messageId:       *res.Messages[0].MessageID,
```

Comment:
One more thing, do we need a nil-check on `MessageID` and the other nullable properties?

Reply:
Done